src/generated/resources/data/gtceu/tags/items/conveyor_modules.json

```diff
@@ -0,0 +1,17 @@
+{
+  "values": [
+    "gtceu:lv_conveyor_module",
+    "gtceu:mv_conveyor_module",
+    "gtceu:hv_conveyor_module",
+    "gtceu:ev_conveyor_module",
+    "gtceu:iv_conveyor_module",
+    "gtceu:luv_conveyor_module",
+    "gtceu:zpm_conveyor_module",
+    "gtceu:uv_conveyor_module",
+    "gtceu:uhv_conveyor_module",
+    "gtceu:uev_conveyor_module",
+    "gtceu:uiv_conveyor_module",
+    "gtceu:uxv_conveyor_module",
+    "gtceu:opv_conveyor_module"
```

Comment:
the high-tier ones need to be marked as optional tag entries so the game won't error.